docs/enabling-and-disabling-applications.md

```diff
@@ -0,0 +1,29 @@
+Enabling and Disabling Applications
+===================================
+
+What is an Applications?
+------------------------
+
+An application is any service natively include with PiNanas (like DHCPD, Traefik, etc).
+All applications are listed and detailled in [README](README.md#settings "README.md")
+
+Why would I want enable/disable some applications?
+---------------------------------------------
+
+PiNanas has been designed to propose all applications "in the box"
+But, you have the possibility to personalise your PiNanas and so
+you will want to be able to select the applications you are interested in.

```

Comment:
PiNanas has been designed to offer all applications "in the box" per default.
You can personalise this behaviour by selecting which applications you are interested in and want to enable.